### DISCUSSION TITLE
Some Issues on 3D Flow Field Inversion

### DISCUSSION THREAD

**Discussion by 15175829310** [mdolab/dafoam#669] [General]
Hello, I am currently conducting flow field inversion on square tubes regarding the issue of secondary flow.
But now I only have DNS data for one cross-section. How can I extract the calculation results of one cross-section for flow field inversion during the inversion process? If possible, how should it be implemented?

**Comment by friedenhe** [comment]
You need to first interpolate DNS data to your RANS mesh and save it as UData. Then, you can extra the cell center coordinates for all the mesh cells on the slice. Then, you can provide those coordinates to the probePoint option. Refer to https://github.com/DAFoam/tutorials/blob/fbc6dcbcbfb006bd21e5e0096e78e66626095194/Ramp/steady/train/runScript_FI.py#L100
